module3/index.md

```diff
@@ -19,26 +19,39 @@ In Module 3, students will begin to dive into the skills and mindsets necessary
 
 ## Curriculum
 
-### Under Construction
-
-<!-- 
-
 ### Week 1
 * How the Web Works (HTTP Request/Response Cycle): [Lesson](./lessons/Week1/HowTheWebWorks) &#124; [Lab](./labs/Week1/HowTheWebWorks)
 * HTML: [Prep](./preparation/Week1/HTMLFundamentals) &#124; [Lesson](./lessons/Week1/HTMLFundamentals) &#124; [Lab](./labs/Week1/HTMLCSSFundamentals)
 * CSS: [Prep](./preparation/Week1/CSSFundamentals) &#124; [Lesson](./lessons/Week1/CSSFundamentals) &#124; [Lab](./labs/Week1/HTMLCSSFundamentals)
 * [Mod 1 & 2 Review](./lessons/Week1/Mod1and2Review)
 
 ### Week 2
-* Feature Testing [Prep](./preparation/Week2/FeatureTesting) &#124; [Lesson](./lessons/Week2/FeatureTesting) &#124; [Lab](./labs/Week2/FeatureTesting) 
+* Intro To MVC: [Prep](./preparation/Week2/IntroToMVC) &#124; [Lesson](./lessons/Week2/IntroToMVC)
```

Comment:
Mod 3 is a thing 🎉 

My only feedback is that I don't particularly like having TODO on student-facing content. What do you think about just not having a link yet for the lessons that are not built yet? We can know that if there is no link there is no content yet?

Reply:
That's ok with me - I was going back and forth because I'm imagining we might get 'this link is broken' messages... but probably not very many :)